support-frontend/app/admin/settings/Switches.scala

```diff
@@ -1,21 +1,121 @@
 package admin.settings
 
 import com.gu.support.encoding.Codec
-import com.gu.support.encoding.Codec.deriveCodec
+import io.circe.generic.extras.Configuration
+import io.circe.generic.extras.auto._
+import io.circe.{Decoder, Encoder, Json}
+import io.circe.optics.JsonPath._
+
+sealed trait SwitchState {
```

Comment:
the `SwitchState` model is more or less as before, but with simpler encoding/decoding